tests/impl.cpp

```diff
@@ -6175,7 +6175,30 @@ result_t test_mm_blend_pd(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_blend_ps(const SSE2NEONTestImpl &impl, uint32_t i)
 {
+#if __x86_64__
```

Comment:
Why did you keep `UNIMPL`?

Reply:
gcc and clang can't compile  call to _mm_blend_ps with 3rd argument as integer type due 4 bit size limitation
and test framework doesn't support compile time constant 
but on target platform its complie , run and passed

Reply:
@marktwtn, is it feasible to extend the existing testing framework for the above considerations?

Reply:
Add the above explanation of excluding x86_64 in the comment of this test case. Later, we might overcome it.

Reply:
comment as is its ok ?

Reply:
BTW 
due its only 4 bits constant 
I can add test with switch case for all possible permutation
if you  prefer it

Reply:
Adding all possible permutations sounds great to me.

Reply:
Done

Reply:
The constant problem is left to be fixed in the future.
However, switch case is a good way to do this.
I will apply the same technique in other tests instead of using fixed constant.